### PR TITLE
Issue 63

### LINKED ISSUE #63: Create enriched output value for the bigip_map
Include the bigip_map data passed into the module and enrich the data with the IP addresses assigned to the various interfaces.

### PR DESCRIPTION
**Pull request by codygreen**
fixes #63

### CODE REVIEW

File: examples/variables.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+variable "cidr" {
+  description = "AWS VPC CIDR"
+  type        = string
+  default     = "10.0.0.0/16"
+}
+
+variable "allowed_mgmt_cidr" {
+  description = "CIDR of allowed IPs for the BIG-IP management interface"
+  type        = string
+  default     = "0.0.0.0/0"
```
Comment (mjmenger) [inline examples/variables.tf:57]: should we consider a more restrictive default for access to the management interface?

File: examples/vpc.tf [hunk 2]
```diff
@@ -0,0 +1,175 @@
+locals {
+  azs  = [format("%s%s", var.region, "a"), format("%s%s", var.region, "b")]
+  cidr = "10.0.0.0/16"
```
Comment (mjmenger) [inline examples/vpc.tf:3]: is there a reason that this isn't pointing to var.cidr?

File: examples/inspec/bigip-ready/inspec.lock [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (mjmenger) [inline examples/inspec/bigip-ready/inspec.lock:1]: should this file be in the .gitignore?

File: examples/variables.tf (hunk 1 above)
Comment (codygreen) [reply examples/variables.tf:57]: @mjmenger, yes but this should probably be it's own issue.

File: examples/vpc.tf (hunk 2 above)
Comment (codygreen) [reply examples/vpc.tf:3]: it should, but again this should be a separate issue

File: examples/inspec/bigip-ready/inspec.lock (hunk 3 above)
Comment (codygreen) [reply examples/inspec/bigip-ready/inspec.lock:1]: yes, but it wasn't created by this PR.

**Pull request merged**